src/test/java/com/epam/university/java/core/task007/Task007Test.java

```diff
@@ -41,4 +42,24 @@ public void testSecond() throws Exception {
                 instance.multiplyPolynomial(first, second)
         );
     }
+
+    @Test
+    public void  testThird() throws Exception {
```

Comment:
I see you're adding new tests to the task but could you please create a separate merge request without your solutions?